config/schemas/config.yaml

```diff
@@ -1923,7 +1923,12 @@ properties:
         additionalProperties: false
         properties:
           disableRedirect:
+            title: Harbor registry disable ObjectStorage redirect
             type: boolean
+            description: |-
+              Controls whether or not Harbor registry redirects users to the ObjectStorage endpoint.
+              Set this to true if the ObjectStorage is not reachable by users when pushing images to Harbor.
```

Comment:
Can you copy-paste the error message that the user would see either in `docker pull`, `docker push` or `kubectl describe pod`?

Reply:
PTAL